cdds/cdds/tests/test_plugins/test_file_info.py

```diff
@@ -303,9 +303,139 @@ def test_get_date_range_subhr_60min(self):
         self.assertEqual(expected_end, end)
 
 
+class TestGlobalModelFileGetRangeGregorian(unittest.TestCase):
+
+    def setUp(self):
+        Calendar.default().set_mode('gregorian')
+        self.model_file_info = GlobalModelFileInfo()
+
+    def test_get_date_range_daily(self):
+        file_template = (
+            '/path/to/output/data/ap6/day/ua/ua_day_dummymodel_dummyexp_dummyvariant_dummygrid_{start}-{end}.nc'
+        )
```

Comment:
Looks like this is common across tests -- worth moving to a class level constant to avoid repetition?